Automattic/woocommerce-payments
# Hide `disputed on` column in disputes listing

Fixes #6661

#### Changes proposed in this Pull Request

Hide `Disputed on` column in the disputes listing.

The changes in #6355 require removing the `disputed on` column. As the CSV is generated using the same code path, removing the column wasn't a straightforward option. The merchant is still able to add the `Disputed on` column back into the report.

The default sort order of the dispute listing is by the `Disputed on` in descending order. As we are hiding this column now it may make sense to update the default sort order to the `Respond by` column. 

![image](https://github.com/Automattic/woocommerce-payments/assets/57298/78c219e7-95bb-42f6-9b35-63e60c0246b1)



#### Testing instructions

1. Visit the Disputes listing page, Payments > Disputes
2. See that the `Disputed On` column is not visible.
3. Download the csv and see that the results are unchanged.
4. Enable the column by using the options menu.

-------------------


**Post merge**
